libminifi/src/core/reporting/SiteToSiteProvenanceReportingTask.cpp

```diff
@@ -68,16 +68,29 @@ void setJsonStr(const std::string& key, const std::string& value, rapidjson::Val
   parent.AddMember(keyVal, valueVal, alloc);
 }
 
-rapidjson::Value getStringValue(const std::string& value, rapidjson::Document::AllocatorType& alloc) { // NOLINT
+rapidjson::Value getStringValue(const std::string& value, rapidjson::Document::AllocatorType& alloc) {
   rapidjson::Value Val;
   Val.SetString(value.c_str(), value.length(), alloc);
   return Val;
 }
 
-void appendJsonStr(const std::string& value, rapidjson::Value& parent, rapidjson::Document::AllocatorType& alloc) { // NOLINT
+template<size_t N>
+rapidjson::Value getStringValue(const utils::SmallString<N>& value, rapidjson::Document::AllocatorType& alloc) {
+  rapidjson::Value Val;
```

Comment:
very minor, but I would rename this variable (and the other `valueVal`s) to `target_value` or `json_value` or something similar to match our naming conventions